ci/authn-k8s/test_oc_entrypoint.sh

```diff
@@ -176,8 +182,8 @@ function loadConjurPolicies() {
   oc exec $conjur_pod -- rake authn_k8s:ca_init["conjur/authn-k8s/minikube"]
 
   # set test password value
-#  password=$(openssl rand -hex 12)
-#  conjurcmd conjur variable values add inventory-db/password $password
+  #password=$(openssl rand -hex 12)
+  #conjurcmd conjur variable values add inventory-db/password $password
```

Comment:
I realize that you just cleaned up the formatting of these lines, but if they were commented out before you started working on it, we should just remove these lines.

Reply:
removed